cli/tools/lint/mod.rs

```diff
@@ -218,7 +220,7 @@ async fn lint_files(
 
   futures.push({
     let has_error = has_error.clone();
-    let linter = create_linter(lint_rules.rules);
```

Comment:
This was only created once. https://github.com/denoland/deno/issues/21876 is out of date and already solved

Reply:
Yeah, sorry @wooseok123, it looks like the previous code below is only creating the linter once for the `run_parallelized`

Reply:
Ah, thank you for checking that. 
I think it needs to add more test code though.
Making a seperate PR for that would be better?